qiskit_cold_atom/circuit_tools.py

```diff
@@ -185,13 +225,71 @@ def circuit_to_cold_atom(
         # validate the circuits against the backend configuration
         cls.validate_circuits(circuits=circuits, backend=backend, shots=shots)
 
+        if "wire_order" in backend.configuration().to_dict():
+            wire_order = backend.configuration().wire_order
+        else:
+            wire_order = cls.__wire_order__
+
         experiments = {}
         for idx, circuit in enumerate(circuits):
             experiments["experiment_%i" % idx] = {
-                "instructions": cls.circuit_to_data(circuit),
+                "instructions": cls.circuit_to_data(circuit, backend=backend),
                 "shots": shots,
                 "num_wires": circuit.num_qubits,
-                "wire_order": cls.__wire_order__,
+                "wire_order": wire_order,
             }
 
         return experiments
+
+    @classmethod
+    def convert_wire_order(
+        cls,
+        wires: List[int],
+        convention_from: str,
+        convention_to: str,
+        num_sites: int,
+        num_species: int,
+        sort: Optional[bool] = False,
```

Comment:
A sequence of indices in one wire convention might not be sorted in the other convention. E.g. `[0, 1, 4, 5]` in "sequential" (4 sites, two spins) goes to `[0, 2, 1, 3]` in "interleafed". I wanted some flexibility because I believe that for some purposes this should be ordered (e.g. checking against the backend coupling map) and for others not.